x/nexus/abci.go

```diff
@@ -68,3 +71,20 @@ func checkChainMaintainers(ctx sdk.Context, n types.Nexus, r types.RewardKeeper,
 
 	return nil
 }
+
+func routeQueuedMessages(ctx sdk.Context, n types.Nexus) {
+	// TODO: make this a module parameter
+	max := 50
+
+	for i := 0; i < max; i++ {
+		msg, ok := n.DequeueRouteMessage(ctx)
+		if !ok {
+			break
+		}
+
+		// try routing the message
+		_ = utils.RunCached(ctx, n, func(ctx sdk.Context) (struct{}, error) {
```

Comment:
That defeats the purpose. We do not want to check anything here but let the router decide whether or not it can be routed.

Reply:
Yeah, wasn't proposing a check here, but that the router avoids panicking instead of returning error due to alerting issues. Looks like that should be the case due to this [check](https://github.com/axelarnetwork/axelar-core/blob/main/x/axelarnet/keeper/message_route.go#L28), so all good